packages/vue/src/components/molecules/SfProductCard/SfProductCard.js

```diff
@@ -54,31 +54,24 @@ export default {
      * It can be a icon name from our icons list, or array or string as SVG path(s).
      */
     wishlistIcon: {
-      type: [String, Array],
+      type: [String, Array, Boolean],
```

Comment:
what is the use case for array?

Reply:
it was for `path` of `sficon` where `path` can be an array of SVG Paths

Reply:
I not sure ai get it. Why would we need an array of paths instead of single one?

Reply:
@filrak  If you look at SfIcon, it received either String as an SVG path or icon name, or an Array of Strings as SVG paths.

Because not all the time the SVG icon has only one path. For example, our add to cart (if you look inside the SVG file, there are paths that formed to an icon.

So Wishlist icon needs to support the same way because we allow the user to customize the icon per user's need